demosplan/DemosPlanCoreBundle/Resources/client/scss/objects/_modal.scss

```diff
@@ -84,4 +84,12 @@
         margin: $inuit-base-spacing-unit 0 0;
         height: auto;
     }
+    &.recommendation-modal {
+        li:hover {
+            background-color: $dp-color-background-light;
+            .badge {
+                background-color: $dp-color-main;
+            }
+        }
+    }
```

Comment:
This must be solved in another way. I guess you're styling the recommendation items here? as the markup is located inside the addon, we must not define styles in core. This would be easy to implement once we have tailwind colors... however for now there is no easy way i guess :(

Reply:
Hm, what would your proposal be in this case? I can't come up with a better idea than either hardcode these styles in a style block for each addon that has these `li` and `badge` classes or to sopy the needed scss (but we would need all those scss variable too in that case?)